GVFS/GVFS.Common/Paths.Shared.cs

```diff
@@ -13,10 +13,24 @@ public static string GetGitEnlistmentRoot(string directory)
 
         public static string GetServiceDataRoot(string serviceName)
         {
-            return Path.Combine(
+            if (Environment.OSVersion.Platform == PlatformID.Win32NT)
+            {
+                return Path.Combine(
                 Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData, Environment.SpecialFolderOption.Create),
                 "GVFS",
                 serviceName);
+            }
+
+            string path = Path.Combine(

```

Comment:
Should we put this in a macOS only block as this will not be right for the future Linux version?

Reply:
Moved `GetServiceDataRoot` to `GVFSPlatform`.